src/databaseInterface/src/main/java/org/vaccineimpact/api/db/annex/Indexes.java

```diff
@@ -40,6 +40,6 @@
     private static class Indexes0 {
         public static Index BURDEN_ESTIMATE_STOCHASTIC_BURDEN_ESTIMATE_SET_IDX = Internal.createIndex("burden_estimate_stochastic_burden_estimate_set_idx", BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC, new OrderField[] { BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.BURDEN_ESTIMATE_SET }, false);
         public static Index BURDEN_ESTIMATE_STOCHASTIC_PKEY = Internal.createIndex("burden_estimate_stochastic_pkey", BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC, new OrderField[] { BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.ID }, true);
-        public static Index BURDEN_ESTIMATE_STOCHASTIC_UNIQUE = Internal.createIndex("burden_estimate_stochastic_unique", BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC, new OrderField[] { BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.BURDEN_ESTIMATE_SET, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.MODEL_RUN, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.COUNTRY, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.YEAR, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.AGE, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.BURDEN_OUTCOME }, true);
+        public static Index BURDEN_ESTIMATE_STOCHASTIC_UNIQUE = Internal.createIndex("burden_estimate_stochastic_unique", BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC, new OrderField[] { BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.BURDEN_ESTIMATE_SET, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.COUNTRY, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.YEAR, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.AGE, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.BURDEN_OUTCOME }, true);
```

Comment:
We should think about the ordering on the unique constraint - although JOOQ may be getting it wrong, so the migration is the thing to examine